### PR TITLE
Update Winapp2.ini

### PR DESCRIPTION
**Pull request by decofona**
[MSI Remind Manager *]
- New Entry

[Adobe Creative Cloud Experience *]
-New Entry

[LibreOffice *]
-FileKey 6 has been added.

[Adobe CC *]
-Detect2 has been added, since it wasn't recognized.

[SteelSeries *]
- New Added

[DSDCS InputMapper *]
- New Added

[JDownloader2 *]
- Actually JDownloader 2.0 will by defaul install into LocalAppData. 
- But I don't know, if earlier versions distributed had a different path.
- So both have been added.
- Also added output.log and install4.j.logs

[Atheros Driver *]
-Changed Detect, since it was not recognizing and added Installog.txt (FileKey1)

### CODE REVIEW

File: Winapp2.ini [hunk 1]
```diff
@@ -2709,6 +2710,14 @@ FileKey17=%Documents%\Adobe\Adobe Media Encoder\*\logs|*.*|RECURSE
...
 RegKey1=HKCU\Software\Adobe\MediaBrowser\MRU
 
+[Adobe Creative Cloud Experience *]
+LangSecRef=3023
+DetectFile=%ProgramFiles%\Adobe\Adobe Creative Cloud Experience
+Default=False
+FileKey1=%ProgramFiles%\Adobe\Adobe Creative Cloud Experience\js\node_modules\table-parser\test\output|*.log
+FileKey2=%ProgramFiles%\Common Files\Adobe\Creative Cloud Libraries\js\node_modules\table-parser\test\output|*.log
+FileKey3=%ProgramFiles%\Common Files\Adobe\Creative Cloud Libraries\js\node_modules\restify|*.log
+
```
Comment (APMichael) [inline Winapp2.ini]: Please use for "%ProgramFiles%\Common Files\" the variable %CommonProgramFiles%.

File: Winapp2.ini [hunk 2]
```diff
@@ -3746,9 +3755,9 @@ FileKey4=%SystemDrive%|wifi.log
 
 [Atheros Driver *]
 LangSecRef=3021
-Detect=HKLM\Atheros
+Detect=HKLM\SOFTWARE\Atheros
 Default=False
-FileKey1=%CommonAppData%\Atheros|*.log|RECURSE
+FileKey1=%CommonAppData%\Atheros|*.log;InstallLog.txt|RECURSE
 FileKey2=%CommonAppData%\Qualcomm Atheros WiFi Driver Installation|*.Log|RECURSE
```
Comment (APMichael) [inline Winapp2.ini]: In the previous pull request it was "%ProgramFiles%\\Common Files\\Atheros|InstallLog.txt" = "%CommonProgramFiles%\\Atheros|InstallLog.txt".

What is correct? %CommonAppData% or %CommonProgramFiles%?

File: Winapp2.ini [hunk 3]
```diff
@@ -10176,11 +10191,18 @@ LangSecRef=3022
...
 FileKey4=%ProgramFiles%\JDownloader*2*\logs|*.*|RECURSE
 FileKey5=%ProgramFiles%\JDownloader*2*\tmp|*.*|RECURSE
+FileKey6=%ProgramFiles%\JDownloader*2*\.install4j|*.log
+FileKey7=%LocalAppData%\JDownloader*2*|JDownloader.log;Updater_*-*-*.log;SessionInstallLog.json.*;output.log|RECURSE
+FileKey8=%LocalAppData%\JDownloader*2*\captchas|*.*|RECURSE
+FileKey9=%LocalAppData%\JDownloader*2*\cfg|CaptchaDialogDimensions_*.json
+FileKey10=%LocalAppData%\JDownloader*2*\logs|*.*|RECURSE
+FileKey11=%LocalAppData%\JDownloader*2*\tmp|*.*|RECURSE
+FileKey12=%LocalAppData%\JDownloader*2*\.install4j|*.log
 ExcludeKey1=PATH|%ProgramFiles%\JDownloader*2*\cfg\|subconf_*.ejs
```
Comment (APMichael) [inline Winapp2.ini]: Please always use Winapp2ool > WinappDebug to make sure the syntax and sorting is correct before opening a new pull request. Thank you very much.

**Comment by decofona** [comment]
Okay Now they have been automatically corrected and sorted. Please check again for both files.

**Comment by APMichael** [comment]
I'm sorry, but it looks like you are using the release version of Winapp2ool. Please use the current beta version as the release version contains a sorting error and does not recognize "Edge Chromium" and "Opera".

https://github.com/MoscaDotTo/Winapp2/tree/Branch1/winapp2ool/bin/Debug

**Comment by decofona** [comment]
@APMichael Okay will use and come back to you.

**Comment by decofona** [comment]
Now look again at both files.

**Comment by APMichael** [comment]
Thank you. At first glance, it looks good now. I will review the changes tomorrow.

**Comment by decofona** [comment]
Okay thanks!

**Comment by APMichael** [comment]
You use LangSecRef=3023 for your new entries [DSDCS InputMapper *], [MSI Remind Manager *] and [SteelSeries *].

Category 3023 (=Multimedia) is audio/video related. Aren't the new entries more Utilities (=3024)?

**Comment by decofona** [comment]
@APMichael You were right, I totally forgot the category when adding. Now it has been corrected.

**Comment by APMichael** [comment]
Thank you very much for your reply and the correction. I will merge the pull request n the next few days.

**Pull request merged**